lambdas/journeyengine/src/main/java/uk/gov/di/ipv/core/journeyengine/JourneyEngineHandler.java

```diff
@@ -185,10 +181,10 @@ private JourneyEngineResult executeJourneyEvent(
                     builder.setJourneyResponse(new JourneyResponse(criStartUri + KBV_CRI_ID));
                     break;
                 case CRI_KBV:
-                    if (journeyStep.equals(NEXT_STEP)) {
+                    if (journeyStep.equals(JourneyStep.NEXT.toString())) {
                         updateUserState(IPV_SUCCESS_PAGE, ipvSessionItem);
                         builder.setPageResponse(new PageResponse(IPV_SUCCESS_PAGE.value));
-                    } else if (journeyStep.equals(ERROR_STEP)) {
+                    } else if (journeyStep.equals(JourneyStep.ERROR.toString())) {
```

Comment:
For KBV, we should add a fail branch but it has its own error page. It's being added here: https://github.com/alphagov/di-ipv-core-front/pull/197

Reply:
Done